src/containers/my-resources/add-resource-modal/AddResourceModal.tsx

```diff
@@ -94,17 +97,29 @@ const AddResourceModal = <T extends TableItem>({
         {...props}
       />
 
-      <Box sx={styles.buttonsArea}>
-        <AppButton onClick={closeModal} variant={ButtonVariantEnum.Tonal}>
-          {t('common.cancel')}
-        </AppButton>
-        <AppButton
-          disabled={!selectedRows.length}
-          onClick={onAddItems}
-          sx={styles.addButton}
-        >
-          {t('common.add')}
-        </AppButton>
+      <Box sx={showCheckboxWithTooltip ? styles.formControls : {}}>
+        {showCheckboxWithTooltip && (
+          <CheckboxWithTooltip
+            label={t('myResourcesPage.resourceDuplication.description', {
+              resource: t(
+                `myResourcesPage.resourceDuplication.resource.${resourceTab}`
+              )
+            })}
+            tooltipTitle={t('myResourcesPage.resourceDuplication.tooltip')}
+          ></CheckboxWithTooltip>
```

Comment:
We can use self-closing component syntax here

Reply:
Thanks! It's already fixed 👌🏻